File: src/main/java/com/gmail/borlandlp/minigamesdtools/localizer/text/LocalizerTextStorage.java
package com.gmail.borlandlp.minigamesdtools.localizer.text;

public interface LocalizerTextStorage {
    /**
     * Проверяет, содержит ли текст для textId
     * @param textId идентификатор текста
     * @return true/false
     */
    boolean containsText(String textId);

    /**
     * Возвращает текст для textId
     * @param textId
     * @return
     */
    String getText(String textId);

    String getLangCode();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/localizer/text/LocalizerTextStorage.kt
package com.gmail.borlandlp.minigamesdtools.localizer.text

interface LocalizerTextStorage {
    /**
     * Проверяет, содержит ли текст для textId
     * @param textId идентификатор текста
     * @return true/false
     */
    fun containsText(textId: String): Boolean

    /**
     * Возвращает текст для textId
     * @param textId
     * @return
     */
    fun getText(textId: String): String?

    val langCode: String?
}